wolfcrypt/src/aes.c

```diff
@@ -3929,7 +3947,7 @@ int wc_AesSetIV(Aes* aes, const byte* iv)
 
     #ifdef WOLF_CRYPTO_CB
         if (aes->devId != INVALID_DEVID) {
-            int ret = wc_CryptoCb_AesCbcEncrypt(aes, out, in, sz);
+            ret = wc_CryptoCb_AesCbcEncrypt(aes, out, in, sz);
```

Comment:
actually it can't be reset -- the static analyzers wail away about unused assigned values -- but I've refactored to use `ret`s that are local to the indented span they're used in, for clarity.